src/MSIdentityScaffolding/Microsoft.DotNet.MSIdentity/Tool/Commands.cs

```diff
@@ -5,11 +5,11 @@ public class Commands
         public const string LIST_AAD_APPS_COMMAND = "--list-aad-apps";
         public const string LIST_SERVICE_PRINCIPALS_COMMAND = "--list-service-principals";
         public const string LIST_TENANTS_COMMAND = "--list-tenants";
-        public const string ADD_CLIENT_SECRET = "--add-client-secret";
         public const string REGISTER_APPLICATIION_COMMAND = "--register-app";
-        public const string UPDATE_APPLICATION_COMMAND = "--update-app-registration";
-        public const string UPDATE_PROJECT_COMMAND = "--update-project";
         public const string UNREGISTER_APPLICATION_COMMAND = "--unregister-app";
-        public const string VALIDATE_APP_PARAMS_COMMAND = "--validate-app-params";
+        public const string ADD_CLIENT_SECRET = "--create-client-secret";
```

Comment:
The initial tool was deducing if there was an update or a creation based on passing the --client-id.

Reply:
--register-app will still do that and that will be the customer facing cmd for the cli experience.